setup-compose.sh

```diff
@@ -5,8 +5,5 @@ else
     export RELEASE="latest"
 fi
 
-## TODO: This should not build the images by default, but should just download them
```

Comment:
@mohamadaminkarami Could we add an option (or a copy of the script) that rebuilds the images? That would be good for debugging in case we need it at some point. I guess to do that we would need to add a different docker-compose that uses the local images instead? Thank you!